HetDepot/Program.cs

```diff
@@ -2,31 +2,16 @@
 {
     internal class Program
     {
-        static void Main(string[] args)
+        static void Main(string[] arg)
 
         
         {
-            Console.WriteLine("Voer uw code in:");
-            int code = Convert.ToInt32 (Console.ReadLine());
-
-
-
-        //     Console.WriteLine ("Voer uw code in:");
-        //     var CodeNumber = Console.ReadLine();
-
-        //     int code;
-        //     if (!int.TryParse(CodeNumber, out code))
-
-        //     {
-        //         Console.WriteLine ("Deze code is niet juist, probeer het nog een keer");
-        //         CodeNumber = Console.ReadLine();
-        //     }
-        //     else
-
-        // Console.WriteLine ("Bedankt!");
-            
-            
+            Console.WriteLine("Hello World");
             
+
+EnterCode.AskforCode();
```

Comment:
Dit is niet heel DRY https://en.wikipedia.org/wiki/Don%27t_repeat_yourself
Ook sluit ik mij aan bij Ruben, kijk of je kunt aansluiten op een controle, en ook verificatie kunt doen.